cityofaustin/joplin
# Get themes out of base/add fixtures

# Description
this does a couple things:
first, you may have noticed the merge-into branch is a weird one, that's because I went ahead and pulled in all the code from https://github.com/cityofaustin/joplin/pull/659 and https://github.com/cityofaustin/joplin/pull/642 before starting on this, so this is an approximation of what it'll look like once those get merged in.

I also moved themes from base to their own app, which made migrations pretty angry, so I rebuilt them.

The big change here is we now have a service page fixture that has a topic/tc/theme line








# Types of changes



# How can this be tested?



 
go here http://joplin-pr-fixtures-for-janis.herokuapp.com/admin/pages/search/
see "kitchen sink" pages

# Checklist:
